src/DropdownTranslation.php

```diff
@@ -481,11 +481,15 @@ public static function showTranslations(CommonDropdown $item)
      */
     public function showForm($ID = -1, array $options = [])
     {
+        if (!isset($options['parent']) || !($options['parent'] instanceof CommonDBTM)) {
```

Comment:
Sounds like a logic change here; and we'll never know why nothing is displayed. At least, an warning message should be emitted in logs

Reply:
`$item` is always used on lines 507/508, so method would have trigger a fatal error if call without a valid object in `$options['parent']`.

I added a `trigger_error` to emit something in logs.